packages/ui/src/lib/toaster/Toaster.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts" context="module">
+	type ToasterPositionClass = {
+		[key: string]: string;
+	};
+
+	const positionClasses: ToasterPositionClass = {
+		TopLeft: 'top-6 left-6',
+		TopCenter: 'top-6 left-1/2 -translate-x-1/2 transform',
+		TopRight: 'top-6 right-6',
+		CenterRight: 'top-1/2 -translate-y-1/2 right-6 transform',
+		BottomRight: 'bottom-6 right-6',
+		BottomCenter: 'bottom-6 left-1/2 -translate-x-1/2 transform',
+		BottomLeft: 'bottom-6 left-6',
+		CenterLeft: 'top-1/2 -translate-y-1/2 left-6 transform',
+		Center: 'top-1/2 -translate-y-1/2 left-1/2 -translate-x-1/2 transform'
+	};
+</script>
+
+<script lang="ts">
+	import Toast from './Toast.svelte';
+	import { messages } from './toaster';
+
+	// position is used to layout the Toaster. You can specify your own classes
+	// for positioning via the classes property if you want something bespoke.
+	export let position = 'TopCenter';
```

Comment:
```suggestion
	import { messages } from './toaster';
	import type { ToasterPosition } from './types';

	// position is used to layout the Toaster. You can specify your own classes
	// for positioning via the classes property if you want something bespoke.
	export let position: keyof typeof ToasterPosition = 'TopCenter';
```

Generating a string union of the values of the enum means auto-complete will work for values of the `position` prop when  using the component.